src/azul/transformer.py

```diff
@@ -42,6 +45,64 @@ class Document:
     version_type: ClassVar[Optional[str]] = None
     contents: Optional[JSON]
 
+    @staticmethod
+    def _translate_field(value, path, forward=True):
+        """
+        A helper function to `translate_values`, this function returns a single translated value
+        :param value: A single field value
+        :param path: A tuple containing the path of the field
+        :param forward: A boolean to indicate if we should translate forward or backward (aka un-translate)
+        :return: The translated/un-translated value, or the original value if no translation performed
+        """
+        # Optional boolean fields: translate (None,False,True) to (-1,0,1) or vice versa
+        if (
+            path[-3:] == ('projects', 'contributors', 'corresponding_contributor') or
+            path[-2:] == ('protocols', 'paired_end')
+        ):
+            if forward:
+                value = {None: -1, False: 0, 'False': 0, True: 1, 'True': 1}[value]
```

Comment:
Function `test_derived_files()` uses canned bundle d5e01f9d-615f-4153-8a56-f2317d7d9ce8. This bundle has a `paired_end` field with string value `'True'` instead of boolean value `True`.
```
    "sequencing_protocol_0.json": {
        "paired_end": "True",
```
This is the only instance I can find where `paired_end` is not a boolean or `None`.